### DISCUSSION TITLE
Seria correto usar a tag <article> para representar a parte hero de uma página?

### DISCUSSION THREAD

**Discussion by PedroBicudo** [frontendbr/forum#2160] [Perguntas e Respostas]
A definição do ```<article>``` diz que ele é um bloco de conteúdo que pode existir em qualquer contexto. Mas a minha dúvida é: Eu posso representar uma área hero usando essa tag?

**Comment by ninetails** [comment]
Semanticamente esta tag surgiu creio que por artigos, como artigo de blog... Porém a definição tanto em português quanto em inglês pode também ser apontada como um ítem específico.

Um ótimo lugar de referência que eu acho sempre foi a [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article):

> The &lt;article&gt; HTML element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, **an interactive widget or gadget, or any other independent item of content.**

O mesmo parágrafo foi retirado da [documentação da W3C](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-article-element.html) (apesar da página estar como _Working Draft_ de 2011, foi a primeira página que veio no Google).

No próprio _demo_ da MDN foi exemplificado um componente de clima...

Então creio que possa sim utilizar como `<article>`, mas creio que daí teria que fazer o conteúdo interno do Hero também semântico, já que também diz que o que a tag aceita dentro seria um "flow content" (conteúdo de fluxo).

(pelo menos foi o que entendi)

**Comment by nazarepiedady** [comment]
Use section ou uma div.
